docs/gettingstarted/index.md

```diff
@@ -6,46 +6,33 @@ This is a simple guide to get you from nothing to a basic mod. The rest of this
 
-[home directory]: https://en.wikipedia.org/wiki/Home_directory#Default_home_directory_per_operating_system "Default user's home folder location for different operation systems"
-[files]: https://files.minecraftforge.net "Forge Files distribution site"
+### IntelliJ IDEA Configuration
 
-Terminal-free IntelliJ IDEA configuration
-------------------------------------------
+5. Launch IntelliJ and choose to open/import the `build.gradle` file, using the default gradle wrapper choice. While you wait for this process to finish, you can open the gradle panel, which will get filled with the gradle tasks once importing, and indexing, is completed.
+6. You can now generate the IntelliJ specific run configurations, based on the `runs` section in the `build.gradle`, for launching the Minecraft client and server. This can be done in either of the following ways:
+    * Using IntelliJ's built-in gradle features - simply to right of the screen `Gradle > <Project Name> > Tasks > fg_runs > genIntellijRuns`.
+    * Open up a terminal, Command Prompt or PowerShell in Windows, making sure the current working directory is the folder created in step 3, and running the `gradlew genIntellijRuns` command.
 
-These instructions assume that you have created the project folder as described in the steps 1 to 3 of the section above. Because of that, the numbering starts at 4.
+### Eclipse Configuration
 
-4. Launch IDEA and choose to open/import the `build.gradle` file, using the default gradle wrapper choice. While you wait for this process to finish, you can open the gradle panel, which will get filled with the gradle tasks once importing is completed.
-5. Run the `setupDecompWorkspace` task (inside the `forgegradle` task group). It will take a few minutes, and use quite a bit of RAM. If it fails, you can add `-Xmx3G` to the `Gradle VM options` in IDEA's gradle settings window, or edit your global gradle properties.
-6. Once the setup task is done, you will want to run the `genIntellijRuns` task, which will configure the project's run/debug targets. 
-7. After it's done, you should click the blue refresh icon **on the gradle panel** (there's another refresh icon on the main toolbar, but that's not it). This will re-synchronize the IDEA project with the Gradle data, making sure that all the dependencies and settings are up to date.
-8. Finally, assuming you use IDEA 2016 or newer, you will have to fix the classpath module. Go to `Edit configurations` and in both `Minecraft Client` and `Minecraft Server`, change `Use classpath of module` to point to the task with a name like `<project>_main`.
+5. Open up a terminal, Command Prompt or PowerShell in Windows, and make sure the current working directory is the folder created in step 3.
+6. Run the `gradlew eclipse` command - this will download the required artifacts for building eclipse projects.
```

Comment:
This is incorrect, the project can be imported directly as an "Existing Gradle Project"